src/serial.rs

```diff
@@ -403,6 +458,7 @@ hal! {
         |remap| remap == 1,
         APB2
     ),
+    #[doc(hidden)]
```

Comment:
I don't like hiding the docs for the other devices. A comment similar to that for USART1 would be better imo